docker-compose.yml

```diff
@@ -164,15 +164,44 @@ services:
       LOG_LEVEL: error
       IS_CHALLENGER: 'true'
       TRANSACTIONS_PER_BLOCK: ${TRANSACTIONS_PER_BLOCK:-2}
-    command: [ 'npm', 'run', 'dev' ]
+    command: ['npm', 'run', 'dev']
+
+  deployer-proposer:
+    image: ghcr.io/eyblockchain/nightfall3-deployer-proposer:latest
+    networks:
+      - nightfall_network
+    environment:
+      OPTIMIST_HOST: optimist1
+      OPTIMIST_WS_PORT: 8080
+      OPTIMIST_HTTP_PORT: 80
+      BLOCKCHAIN_WS_HOST: blockchain1
+      BLOCKCHAIN_PORT: 8546
+      CLIENT_HOST: client1
+      DEPLOYER_HOST: deployer
+      TEST_BLOCKS: 2
+
+  genesis-blocks-deployer:
+    image: ghcr.io/eyblockchain/nightfall3-genesis-blocks-deployer:latest
+    networks:
+      - nightfall_network
+    environment:
+      OPTIMIST_HOST: optimist1
+      OPTIMIST_WS_PORT: 8080
+      OPTIMIST_HTTP_PORT: 80
+      CLIENT_HOST: client1
+      DEPLOYER_HOST: deployer
+      CLIENT_PORT: 80
+      BLOCKCHAIN_WS_HOST: blockchain1
+      BLOCKCHAIN_PORT: 8546
+      IS_TEST_RUNNER: 'yes'
+      TEST_LENGTH: 1
+      TRANSACTIONS_PER_BLOCK: ${TRANSACTIONS_PER_BLOCK:-2}
```

Comment:
This is received from config to the user. Is there a reason why we added here?

Reply:
Why is this resolved if nothing was removed or a comment was not provided about why it needs not be resolved?